app/routes/friends.js

```diff
@@ -2,13 +2,15 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
   actions: {
-    save: function() {
+    save() {
       return true;
     },
-    cancel: function() {
+
+    cancel() {
       return true;
     },
-    delete: function(friend) {
+
+    delete(friend) {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).